portfolio/src/main/webapp/scriptMaps.js

```diff
@@ -0,0 +1,177 @@
+            },
+            {
+                featureType: 'poi',
+                elementType: 'labels.text.fill',
+                stylers: [{ color: '#d59563' }]
+            },
+            {
+                featureType: 'poi.park',
+                elementType: 'geometry',
+                stylers: [{ color: '#263c3f' }]
+            },
+            {
+                featureType: 'poi.park',
+                elementType: 'labels.text.fill',
+                stylers: [{ color: '#6b9a76' }]
+            },
+            {
+                featureType: 'road',
+                elementType: 'geometry',
+                stylers: [{ color: '#38414e' }]
+            },
+            {
+                featureType: 'road',
+                elementType: 'geometry.stroke',
+                stylers: [{ color: '#212a37' }]
+            },
+            {
+                featureType: 'road',
+                elementType: 'labels.text.fill',
+                stylers: [{ color: '#9ca5b3' }]
+            },
+            {
+                featureType: 'road.highway',
+                elementType: 'geometry',
+                stylers: [{ color: '#746855' }]
+            },
+            {
+                featureType: 'road.highway',
+                elementType: 'geometry.stroke',
+                stylers: [{ color: '#1f2835' }]
+            },
+            {
+                featureType: 'road.highway',
+                elementType: 'labels.text.fill',
+                stylers: [{ color: '#f3d19c' }]
+            },
+            {
+                featureType: 'transit',
+                elementType: 'geometry',
+                stylers: [{ color: '#2f3948' }]
+            },
+            {
+                featureType: 'transit.station',
+                elementType: 'labels.text.fill',
+                stylers: [{ color: '#d59563' }]
+            },
+            {
+                featureType: 'water',
+                elementType: 'geometry',
+                stylers: [{ color: '#17263c' }]
+            },
+            {
+                featureType: 'water',
+                elementType: 'labels.text.fill',
+                stylers: [{ color: '#515c6d' }]
+            },
+            {
+                featureType: 'water',
+                elementType: 'labels.text.stroke',
+                stylers: [{ color: '#17263c' }]
+            }
+        ], { name: 'Dark Mode' });
+
+    var chelseaLatLng = { lat: 42.318, lng: -84.020 };
+    var hagueLatLng = { lat: 52.0705, lng: 4.3007 };
+
+    var chelseaString = '<h1>Chelsea, MI</h1>' + 
+        '<div><p>Chelsea is a city in Washtenaw County in the U.S.' +
+        ' state of Michigan. The population was 4,944 at the 2010 census.</p></div>' +
+        '<p>Attribution: Chelsea, ' +
+        '<a href="https://en.wikipedia.org/wiki/Chelsea,_Michigan">' +
+        'https://en.wikipedia.org/wiki/Chelsea,_Michigan</a> (last visited July 07, 2020).</p>' +
+        '</div></div>';
```

Comment:
These two `/div`s don't seem to actually be closing anything.

Also, I'm not sure why you have a div at all in here, since you're not applying a style. The "Chelsea is a..." sentence can just be in a \<p\> block.